File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s= s.strip()
        count =0
        
        for i in range(len(s)-1, -1, -1):
            if s[i]==" ":
                break
            else:
                count+=1
                
            
        return count
        

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        s = s.trim();
        int i = s.length()-1;
        int counter = 0;
        while (i>=0 && s.charAt(i) !=' '){
            counter++;
            i--;
        }
        return counter;
    }
}